Sources/Rosetta/CardSets/Expert1CardsGen.cpp

```diff
@@ -3689,6 +3699,18 @@ void Expert1CardsGen::AddNeutralNonCollect(std::map<std::string, Power>& cards)
     power.AddEnchant(new Enchant(Enchants::AddAttackScriptTag));
     cards.emplace("NEW1_018e", power);
 
+    // ---------------------------------- ENCHANTMENT - NEUTRAL
+    // [NEW1_024o] Greenskin's Command (*) - COST:0
+    // - Set: Expert1
+    // --------------------------------------------------------
+    // Text: +1/+1.
+    // --------------------------------------------------------
+    power.ClearData();
+    std::vector<Effect*> enchantEffects = { Effects::AttackN(1),
+                                            Effects::DurabilityN(1) };
+    power.AddEnchant(new Enchant(enchantEffects));
```

Comment:
I think that this statements can be combined.

Reply:
fixed.